djangocms_versioning_filer/cms_config.py

```diff
@@ -39,6 +57,9 @@ def on_file_unpublish(version):
     )
     file_content.file = move_file(file_content, path)
     file_content.save()
+    
+    if type(file_content) == Image:
```

Comment:
this reads like it tries to remove the thumbnails after the unpublish which implies that the previously published thumbnails are still around. did this work?

Reply:
@kinkerl , I have tested in both `publish` and `unpublish` cases, and they work.